plugins/woocommerce-admin/client/two-column-tasks/headers/purchase.js

```diff
@@ -0,0 +1,47 @@
+/**
+ * External dependencies
+ */
+import { Button } from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import TimerImage from './timer.svg';
+import { WC_ASSET_URL } from '../../utils/admin-settings';
+
+const PurchaseHeader = ( { task, goToTask } ) => {
+	return (
+		<div className="woocommerce-task-header__contents-container">
+			<img
+				alt={ __( 'Purchase illustration', 'woocommerce' ) }
+				src={
+					WC_ASSET_URL + 'images/task_list/purchase-illustration.png'
+				}
+				className="svg-background"
+			/>
+			<div className="woocommerce-task-header__contents">
+				<h1>{ task.title }</h1>
+				<p>
+					{ __(
+						'Good choice! You chose to add amazing new features to your store. Continue to checkout to complete your purchase.',
+						'woocommerce'
+					) }
+				</p>
+				<Button
+					isSecondary={ task.isComplete }
+					isPrimary={ ! task.isComplete }
+					onClick={ goToTask }
```

Comment:
This should actually do something different, as it navigates to the `purchase` task screen which doesn't exist.
It should show a popup like this instead:
<img width="986" alt="Screen Shot 2022-06-01 at 4 07 39 PM" src="https://user-images.githubusercontent.com/2240960/171482961-33d7b2d5-144a-4d84-ac2f-7aed63200451.png">